### ISSUE TITLE
Cannot find symbol method toInstant()

### ISSUE THREAD

**Issue by eugenkiss** [requery/requery#6]
Hey,

I've cloned this repository and was trying to run the example project via Android Studio. However, I received the following error message: cannot find symbol method toInstant(). Here's a screenshot of the compilation errors.
![screen shot 2016-02-19 at 09 36 58](https://cloud.githubusercontent.com/assets/211365/13170517/d7a5a082-d6ec-11e5-9a63-053eada7f514.png)

Do you have any pointers about how to solve this problem?

**Comment by npurushe** [comment]
Do you have Oracle JDK 8 set as your JDK? I would suspect that is the problem If you're building the project you need to use JDK 8 as the project supports Java 8 features when used by regular Java projects.

**Comment by eugenkiss** [comment]
Thanks for your pointer. In fact, I had JDK 8 set as my project JDK and OSX's Java Control Panel suggested it was the version corresponding to jdk8u73. Entering `java -version` in the terminal had the following output:

    java version "1.8.0"
    Java(TM) SE Runtime Environment (build 1.8.0-b132)
    Java HotSpot(TM) 64-Bit Server VM (build 25.0-b70, mixed mode)

I googled for "cannot find symbol method toInstant" and [this was the first result](http://stackoverflow.com/questions/22784030/jdk8-with-source-1-7-default-methods) hinting at a bug in the Java 8 compiler that should however have been resolved already. So I assumed I still had an old JDK version installed and downloaded jdk8u74 from [Oracle's download page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html). I updated the JDK location in the project structure to point from `/Library/Java/JavaVirtualMachines/jdk1.8.0.jdk/Contents/Home` to `/Library/Java/JavaVirtualMachines/jdk1.8.0_74.jdk/Contents/Home` and after that everything worked.

Anyway, it works fine now ;).
